src/index.html

```diff
@@ -11,7 +11,167 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="POST" 
+      autocomplete="off">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div>
+          <label class="form-field">
+            Surname: <input type="text" name="surname">
+          </label>
```

Comment:
```suggestion
          <label class="form-field">
            Surname: 
            <input type="text" name="surname">
          </label>
```
Please fix all cases